src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -90,12 +90,7 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
             BindToOption(DisplayLineSeparators, FeatureOnOffOptions.LineSeparator, LanguageNames.CSharp);
 
             // Quick Actions
-            BindToOption(ComputeQuickActionsAsynchronouslyExperimental, SuggestionsOptions.Asynchronous, () =>
-            {
-                // If the option has not been set by the user, check if the option is enabled from experimentation.
-                // If so, default to that.
-                return optionStore.GetOption(SuggestionsOptions.AsynchronousFeatureFlag);
```

Comment:
Would you still want to bind this to the disabling flag? If (for some very unexpected reason) we do have to disable this via the feature flag, I'm guessing you don't want the UI to be checked? Or at this point, if we consider this non-experimental, why have the checkbox at all?

I'm also not sure under what scenario somebody would turn this off, other than we discover a general bug, at which point are we turning it off for everybody?

Reply:
(should we just remove the checkbox?)

Reply:
YEah, i'll remove.